test/unit/test_pdb.c

```diff
@@ -149,13 +185,13 @@ bool test_pdb_tpi_cpp(void) {
 			mu_assert_eq(result || stype, 0, "wrong class vshape");
 			result = type_info->get_derived(type_info, (void **)&stype);
 			mu_assert_eq(result || stype, 0, "wrong class derived");
-		} else if (type->tpi_idx == 0x1062) {
-			mu_assert_eq(type_info->leaf_type, eLF_BITFIELD, "Incorrect data type");
-			SType *base_type = NULL;
-			type_info->get_base_type(type_info, (void **)&base_type);
-			char *type;
-			type_info->get_print_type(type_info, &type);
-			mu_assert_streq(type, "bitfield uint32_t : 1", "Incorrect bitfield print type");
+			//} else if (type->tpi_idx == 0x1062) {
+			//	mu_assert_eq(type_info->leaf_type, eLF_BITFIELD, "Incorrect data type");
+			//	SType *base_type = NULL;
+			//	type_info->get_base_type(type_info, (void **)&base_type);
+			//	char *type;
+			//	type_info->get_print_type(type_info, &type);
+			//	mu_assert_streq(type, "bitfield uint32_t : 1", "Incorrect bitfield print type");
```

Comment:
I commented this since the test was wrong - `bitfield uint32_t : 1` is a wrong syntax and handling bitfields in PDB should be done differently.